framework/mesh/mesh_matrix3x3.h

```diff
@@ -5,28 +5,17 @@
 
 #include <cmath>
 #include <sstream>
+#include <array>
+#include <algorithm>
 #include "framework/mesh/mesh_vector.h"
 
 namespace opensn
 {
 
 struct Matrix3x3
 {
-  double vals[9];
-
-  /// Creates a zeros matrix of size 3x3.
-  Matrix3x3()
-  {
-    vals[0] = 0.0;
-    vals[1] = 0.0;
-    vals[2] = 0.0;
-    vals[3] = 0.0;
-    vals[4] = 0.0;
-    vals[5] = 0.0;
-    vals[6] = 0.0;
-    vals[7] = 0.0;
-    vals[8] = 0.0;
-  }
+  /// Row-major storage for matrix elements
```

Comment:
I don't think this comment is necessary. Or could be shortened to `Storage for matrix elements`.

Reply:
done